apps/i18n/common/en_us.json

```diff
@@ -593,10 +593,10 @@
   "projectTypeGamelabViewMore": "View more Game Lab projects",
   "projectTypePlaylabViewMore": "View more Play Lab projects",
   "projectsViewAll": "View all projects",
-  "projectsViewProjectGallery": "View project gallery",
-  "projectGalleryHeader": "Project Gallery",
+  "projectsViewProjectGallery": "View projects",
   "projectsViewOldGallery": "View old gallery with other project types",
   "projects": "Projects",
+  "projectsSubHeading": "Students have created over 20 million projects",
```

Comment:
Sorry, didn't update TODO list with the copy. The string should be "Over 20 million projects created"

Is it possible to do formatting and somehow make the `20 million` more bold and more big?

Reply:
We can either have a localized string OR we can style the 20 million differently than the rest of the string. My understanding is that for translations to be successful we can't translate substrings and then put them back together because maybe the ordering matters in different languages, etc. What's your preference?

Reply:
Merp. Ok, let's go with localized > formatting.